guvnor-project/guvnor-project-api/src/main/java/org/guvnor/common/services/project/service/ProjectRepositoryResolver.java

```diff
@@ -31,6 +31,8 @@
 
     String CONFLICTING_GAV_CHECK_DISABLED = "org.guvnor.project.gav.check.disabled";
 
+    String CHILD_GAV_EDITION_ENABLED = "org.guvnor.project.gav.child.edit.enabled";
```

Comment:
```CHILD_GAV_EDIT_ENABLED``` would be more in keeping with the System Property and function... but it is me being a bit picky....